agent/taskresource/cgroup/cgroup_unsupported.go

```diff
@@ -0,0 +1,116 @@
+// +build !linux
+// Copyright 2014-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package cgroup
+
+import (
+	"errors"
+	"time"
+
+	"github.com/aws/amazon-ecs-agent/agent/taskresource"
+)
+
+// CgroupResource represents Cgroup resource
+type CgroupResource struct{}
```

Comment:
It should ideally not be. But depending on how the flow of cgroup resource creation is (if its in a common file and task cpu limit var is enabled by mistake, windows would look for cgroup obj), this may or may not be used. The next PR should determine that and we can remove or keep this accordingly.

Reply:
> (if its in a common file and task cpu limit var is enabled by mistake, windows would look for cgroup obj), this may or may not be used.

Even in that case, it should be handled in the config package. i.e. if windows, explicitly disable it. If that's not being enforced, can you please make that change so that this file is not needed at all?

Reply:
This is to confirm that config explicitly disables task cpu limit for windows. Removed `cgroup_unsupported.go`.